src/main/telemetry/smartport.c

```diff
@@ -328,6 +328,35 @@ void handleSmartPortTelemetry(void)
                     smartPortHasRequest = 0;
                 }
                 break;
+            case FSSP_DATAID_CELLS       :
+                if (feature(FEATURE_VBAT) && telemetryConfig->telemetry_flvss_cells) {
+                    /*
+                     * A cell packet is formated this way: https://github.com/jcheger/frsky-arduino/blob/master/FrskySP/FrskySP.cpp
+                     * content    | length
+                     * ---------- | ------
+                     * volt[id]   | 12-bits
+                     * celltotal  | 4 bits
+                     * cellid     | 4 bits
+                     */
+                    static uint8_t currentCell = 0; // Track current cell index number
+                    
+                    // Cells Data Payload
+                    uint32_t payload = 0;
+                    payload |= ((uint16_t)(vbat * 100 + batteryCellCount) / (batteryCellCount * 2)) & 0x0FFF;  // Cell Voltage formatted for payload, modified code from cleanflight Frsky.c, TESTING NOTE: (uint16_t)(4.2 * 500.0) & 0x0FFF;
```

Comment:
2 cell values are sent in single packet, 12bit each

see:
https://github.com/opentx/opentx/blob/master/radio/src/telemetry/frsky_sport.cpp#L197

here is my implementation (rounding of cellVoltage is missing)
https://github.com/ledvinap/cleanflight/blob/local/src/main/telemetry/sport.c#L279

BTW: shifting values into correct position (as in frsky sendVoltage) is much easier to read ...